src/System.Runtime.Serialization.Json/tests/System.Runtime.Serialization.Json.Tests.csproj

```diff
@@ -11,16 +11,21 @@
     <AssemblyName>System.Runtime.Serialization.Json.Tests</AssemblyName>
     <ProjectGuid>{701CB3BC-00DC-435D-BDE4-C5FC29A708A7}</ProjectGuid>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(UseContractReferences)' == 'true'" >
```

Comment:
Yes the project won't build unless we don't set UseContractReferences or we set it to exactly 'true'. If we use ```!= 'true'``` then if the we specifies values like 'True' or 'yes' the project will build but we have different behavior. So I'm just afraid that may cause confusion.